src/content/configuration/watch.md

```diff
@@ -50,6 +50,15 @@ module.exports = {
 };
 ```
 
+__webpack.config.js__
```

Comment:
this example without explanation doesn't make much sense to me, do you want to explain what happens when you pass in a number?